ci/setup_env.sh

```diff
@@ -36,9 +36,17 @@ else
   exit 1
 fi
 
-wget -q "https://repo.continuum.io/miniconda/Miniconda3-latest-$CONDA_OS.sh" -O miniconda.sh
+if [ "${TRAVIS_CPU_ARCH}" == "arm64" ]; then
+  apt-get update && apt-get install xvfb;
```

Comment:
Thanks for the update @ossdev07, much better now.

I see now. Apparently, for the rest of the architectures `xvfb` is installed by default, but not for `aarch64`. I thought Travis would be more consistent, but I see your point now.

I know we asked you to remove it, but can you please restore this in the yaml as `services: xvfb` please? And remove this line.

Also, can you remove the semicolon you added at the end of the lines in this file `CONDA_URL=...`, and `wget ...`.

Reply:
@datapythonista ,

Thanks for the feedback. I will update as suggested.

Reply:
@datapythonista ,

I have updated the commit with changes suggested. Please review.

Reply:
@datapythonista ,

I have updated the commit with changes suggested. Please review.